### ISSUE TITLE
Plugin crashes when trying to create `grafana_contact_point` resource after upgrade to v2.8.0

### ISSUE THREAD

**Issue by ana-cozma** [grafana/terraform-provider-grafana#1258]
### Terraform Version
* Terraform: v1.6.6
* Terraform Grafana Provider: v2.8.0
* Grafana

### Affected Resource(s)
Please list the resources as a list, for example:
* grafana_contact_point

### Terraform Configuration Files
```hcl
resource "grafana_contact_point" "contact_point" {
  name = title("${var.team} Slack")
  slack {
    url = var.slack_webhook_url
  }
}

```

### Debug Output
[Gist Link here](https://gist.github.com/ana-cozma/46f595d45c2c12e0bcfbef855263ee67#file-gistfile1-txt)

### Panic Output
N/A

### Expected Behavior
What should have happened?
The resource is created as expected.

### Actual Behavior
What actually happened?
Error: The terraform-provider-grafana_v2.8.0 plugin crashed!

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Created some resources on v1.25
2. Upgraded Grafana provider to v2.8.0
3. Created a new resource grafana_contact_point with basic Slack integration settings
4. Ran `terraform apply`

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?
No, running on Grafana Cloud.

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
Could not find one

**Comment by jdgeisler** [comment]
After upgrading to v2.8.0 of the grafana tf provider I also began to experience similar issues with contact points. 

My plugin did not crash, but instead when doing a terraform plan over many different contact points I began to experience time outs over every contact point with the below error:

`Error: Get "https://grafana.rokulabs.net/api/v1/provisioning/contact-points": context deadline exceeded`

Downgrading to v2.7.0 fixed the issue, so it seems it might point to a bug introduced by a contact point refactor in v2.8.0.

**Comment by julienduchesne** [comment]
👍 It's already fixed in the main branch. Will be fixed in next release

**Comment by julienduchesne** [comment]
Should be fixed in v2.8.1: https://github.com/grafana/terraform-provider-grafana/releases/tag/v2.8.1
